keyboard/ergodox_ez/keymaps/keymap_j3rn.c

```diff
@@ -0,0 +1,186 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
+
+#define BASE 0 // default layer
+#define SYMB 1 // symbols
+#define MDIA 2 // media keys
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* Keymap 0: Basic layer
+ *
+ * ,--------------------------------------------------.           ,--------------------------------------------------.
+ * | Grv    |   1  |   2  |   3  |   4  |   5  |  [   |           |  ]   |   6  |   7  |   8  |   9  |   0  |  BkSp  |
```

Comment:
Oh, great! For some reason I thought the top-left key isn't supposed to be Grv, I guess I was wrong. :) All good, merging now. Thank now @J3RN!